vault/overlays/nerc-ocp-infra/auto-unseal/scripts/check-vault-seal.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -ue
+
+echo "Checking vault seal status..."
+
+oc exec -i nerc-vault-0 -- vault status --format=json > /tmp/sealed.json
```

Comment:
Not really a problem, but by preference I would use `kubectl` instead of `oc` unless you actually need some sort of openshift-specific feature.

Reply:
Noted.. And I agree, will change to kubectl